dist/sessions.js

```diff
@@ -43,6 +59,87 @@ class Sessions {
       };
     });
   }
+  /** Parse a JWT and verify the signature, preferring local verification over remote.
+   *
+   * If maxTokenAge is set, remote verification will be forced if the JWT was issued at (based on
+   * the "iat" claim) more than maxTokenAge seconds ago.
+   */
+
+
+  async authenticate_jwt(jwt, options) {
+    try {
+      const session = await this.authenticate_jwt_local(jwt, options);
+      return {
+        session,
+        session_jwt: jwt,
+        session_token: "",
+        // Pretend that this was a successful API request.
+        request_id: "",
```

Comment:
Our API will return a `request_id` here, imo we shouldn't overwrite as an empty string.

Reply:
Whoops, I missed this when I was going through the diff view because GH auto-collapses compiled code (`dist` and `types`) set by our `.gitattributes` file. I think this is the generated equivalent of [this](https://github.com/stytchauth/stytch-node/pull/116#discussion_r831672603).